scipy/stats/_qmc.py

```diff
@@ -1543,19 +1550,16 @@ def random(self, n: _int = 1) -> np.ndarray:
             return transf_samples[:, : self.d]
 
 
-class MultivariateNormalQMC(QMCEngine):
-    r"""QMC sampling from a multivariate Normal :math:`N(\mu, \Sigma)`.
+class MultinomialQMC(QMCEngine):
```

Comment:
Maybe add the `reset` method as well. It should call the reset function of the engine.